doc-content/enterprise-only/release-notes/rn-7.10.1-known-issues-ref.adoc

```diff
@@ -0,0 +1,76 @@
+[id='rn-7.10.1-known-issues-ref']
+= Known issues in {PRODUCT} 7.10.1
+
+This section lists known issues with {PRODUCT} 7.10.1.
+
+ifdef::PAM[]
+
+== Process Designer
+
+.Custom task thumbnails are not consistent [https://issues.redhat.com/browse/RHPAM-3496[RHPAM-3496]]
+
+Issue: Custom task thumbnails are not consistent in both project and case project.
+
+Steps to reproduce:
+
+. Create a case project.
+. In case project, create both process and case.
+. Save the changes and close the process.
+. Navigate to the project settings and add all the custom tasks.
+. Save the changes.
+. Check the process.
++
+If the issue is not visible try to close and reopen the asset.
+. Close the process.
+. Remove all the custom tasks and save the changes.
+. Check the process.
++
+If the issue is not visible try to close and reopen the asset.
+
+Expected result: Custom task thumbnails are consistent.
+
+Actual result: Custom task thumbnails are not consistent.
+
+Workaround: None.
+
+.JavaScript language in an *On Entry Action* causes an unexpected system error after changing node to *Multiple Instance* [https://issues.redhat.com/browse/RHPAM-3409[RHPAM-3409]]
+
+Issue: In the *Properties* panel, if the language is set to JavaScript in an *On Entry Action* property and you change the node to *Multiple Instance*, you receive an unexpected system error.
+
+Steps to reproduce:
+
+. Create a new business process.
+. Create a task that contains *Multiple Instance* property.
+. Enter any string to the *On Entry Action* property.
+. Change the language to JavaScript.
+. Set the value of the *Multiple Instance* property to `true`.
+
+Expected result: No errors occur in the user interface or server log.
+
+Actual result: You receive an unexpected system error.
+
+Workaround: None.
+
+== {KIE_SERVER}
+
+.Execution of the `WebServiceWorkItemHandler` running on {EAP} 7.3.5 fails [https://issues.redhat.com/browse/RHPAM-3440[RHPAM-3440]]
+
+Issue: The execution of the `WebServiceWorkItemHandler` on {KIE_SERVER} running on {EAP} 7.3.5 fails. The `ModuleClassLoader` is changed in the latest {EAP} release causes incompatibility with the code working in previous versions.
+
+NOTE: Do not upgrade to {EAP} 7.3.5 if you are using `WebServiceWorkItemHandler` on {KIE_SERVER}. Use {EAP} 7.3.4 instead.
+
+Steps to reproduce:
+
+. Clone the https://github.com/kiegroup/droolsjbpm-integration/tree/master/kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm[droolsjbpm-integration] repository.
+. Go to `droolsjbpm-integration/kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/` directory.
+. Execute the following command:
++
+[source]
+----
+mvn clean install -Peap7 -Deap7.download.url=<eap 7.3.5 zip file URL> -Dit.test=WebServiceIntegrationTest
+----
+Test fails along with the errors in the console.
```

Comment:
[style]

s/Test fails along with the errors in the console./Test fails with errors displayed in the console.